sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/AssetConversionConfiguration.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.MixedReality.ObjectAnchors.Models;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> Represents an asset conversion configuration. </summary>
+    [CodeGenModel("IngestionConfiguration")]
+    public partial class AssetConversionConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AssetConversionConfiguration"/> class.
+        /// </summary>
+        /// <param name="gravity">The asset gravity, which must be normalized.</param>
+        /// <param name="scale">The asset scale.</param>
+        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale) : this(new Vector3(gravity), scale)
+        {
+            if (!gravity.IsNormalized())
+            {
+                throw new ArgumentException("The value must be normalized.", nameof(gravity));
+            }
+        }
+
+        internal AssetConversionConfiguration(Vector3 gravityWrapper, float scale)
+        {
+            if (gravityWrapper == null)
+            {
+                throw new ArgumentNullException(nameof(gravityWrapper));
+            }
+
+            GravityWrapper = gravityWrapper;
+            KeyFrameIndexes = new ChangeTrackingList<int>();
+            GroundTruthTrajectoryCameraPoses = new List<TrajectoryPose>();
+            Scale = scale;
+            TestTrajectoryCameraPoses = new List<TrajectoryPose>();
+        }
+
+        /// <summary> Scale of transformation of asset units into meter space. </summary>
+        public float Scale { get; internal set; }
+        /// <summary> Ground truth trajectory. </summary>
+        [CodeGenMember("GtTrajectory")]
+        public IReadOnlyList<TrajectoryPose> GroundTruthTrajectoryCameraPoses { get; }
+        /// <summary> Dimensions of the asset. </summary>
```

Comment:
Formatting: Please add lines separating the properties. It make it easier to look at.

Reply:
Visual studio makes it difficult to tell if there's a space or not with its "x references" tip right above... fix pending.

Reply:
Properties separated by lines